### ISSUE TITLE
Wrong bond in trajectory

### ISSUE THREAD

**Issue by wangenau** [nglviewer/nglview#1036]
Hi,
I am trying to view the movement of selected particles using nglview and pytraj.
Ideally, I do not want to plot the bonds between these particles.
The problem is, if two particles are really close together in one step and move very far away in the next step there will be a bond drawn between them. I do not think this is intentional.
Is there any way to disable these unwanted bonds, or to disable rendering of bonds altogether?

An (probably uncomplete) list of things I tried:
* Setting nobondsearch in pytraj
* Using noble gases as particles
* bondScaling seems to have no effect
* Changing the identifiers and groups of atoms in the PDB files
* Scaling the coordinates by a lot to increase the distance between atoms

![screenshot](https://user-images.githubusercontent.com/15988797/192231929-b8d7ec35-fb02-49fd-a2bd-b57468aaadea.png)

### Minimal working example
An MWE is found below, using two minimal [PDB files](https://github.com/nglviewer/nglview/files/9644432/mwe_pdb_files.zip)

```bash
import nglview as nv
import pytraj as pt

files = ['file1.pdb', 'file2.pdb']
top = pt.load_topology(files[0], 'nobondsearch')
traj = pt.load(files, top=top)

view = nv.show_pytraj(traj)
view.clear()
view.add_ball_and_stick(f'_He', color='red', radius=1)
view
```

### Code versions
```bash
nglview 3.0.3
ipywidgets 7.7.2
pytraj 2.0.6.dev0
```

**Comment by hainm** [comment]
Dear @wangenau, `nglview` does not support bond breaking like that. `nglview` uses first structure for topology. If you do not want the bond, swap the structures to trick nglview.

`files = ['file2.pdb', 'file1.pdb']`

Here is my guess, I have never tried myself.

**Comment by wangenau** [comment]
Hey, thanks for the quick answer.
Swapping the files does work, as you said.
Sadly, this is not an option for me, since the order of files is important (normally there are a lot more files).
Is there a way to set the file to use for the topology analysis manually?
If so, one could create a mock-pdb file where all atoms are well separated and that does not show op in the trajectory.
The other (not very nice solution) is to remove atoms that are close together.
But this could lead to files being excluded, since the atom count differs.

**Comment by hainm** [comment]
@wangenau hi, you already knew how to do that:

```
top = pt.load_topology(<your desired pdb file>)
traj = pt.load(files, top=top)
```

**Comment by wangenau** [comment]
Oh, I somehow thought that nglview would create an own topology file, since the topology I created didn't have any bonding information saved, due to the 'nobondsearch' keyword. So this did not work for me.
The best solution I got was removing close atoms and find an optimal value such that the trajectory is not disturbed.
I am not super satisfied with this, but I can close the issue if you want to.
But thank you for your help either way!
